packages/suite-desktop-core/e2e/support/common.ts

```diff
@@ -25,9 +25,9 @@ export const launchSuite = async (params: LaunchSuiteParams = {}) => {
         args: [
             path.join(appDir, './dist/app.js'),
             `--log-level=${desiredLogLevel}`,
-            // '--bridge-node-test',
+            '--bridge-node-test',
```

Comment:
I cannot find any results when searching for  `bridge-node-test` in the codebase. Does it do anything?